cmake/README.md

```diff
@@ -0,0 +1,53 @@
+# ESP32CommandStation Cmake project
+
+This directory includes the CMake build system as an alternative to platformio.
+See the general setup/usage instructions 
+ https://docs.espressif.com/projects/esp-idf/en/latest/get-started-cmake/index.html
+ 
+ 
+# Setup
+
+```
+git clone --recurse https://github.com/WolfgangGitHub/DCCppESP32.git
+cd DCCppESP32
```

Comment:
References should be to the main repo and follow the rebranded name.

Reply:
ok - I think removing the line is the easiest solution